test/common/src/main/kotlin/org/codice/compliance/verification/binding/PostBindingVerifier.kt

```diff
@@ -79,20 +121,20 @@ class PostBindingVerifier(private val response: IdpPostResponseDecorator) {
         val relayState = response.relayState
         val givenRelayState = response.isRelayStateGiven
 
-        if (relayState == null) {
-            if (givenRelayState) {
-                throw SAMLComplianceException.create(SAMLBindings_3_4_3_b1, message = "RelayState not found.")
-            }
+        if (!givenRelayState && relayState == null) {
             return
```

Comment:
✏️ I think the logic would read a lot better if in the `verify` method, we just check `givenRelayState` is true before ever calling this method.